common/src/main/java/tachyon/util/network/NetworkAddressUtils.java

```diff
@@ -119,8 +203,20 @@ public static String getLocalIpAddress(int timeout) {
       // Make sure that the address is actually reachable since in some network configurations
       // it is possible for the InetAddress.getLocalHost() call to return a non-reachable
       // address e.g. a broadcast address
-      if (address.isLoopbackAddress() || !address.isReachable(timeout)) {
+      if (address.isAnyLocalAddress() || address.isLoopbackAddress()
```

Comment:
Seems like we're reusing this logic in the if statement on line 227. Could be refactored into a helper method.